Alicerce/Sources/Extensions/Thread.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Thread.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 17/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+extension Thread {
+
+    // Returns the name of the thread or 'main-thread', if it's the application's main thread
+    public class func threadName() -> String {
```

Comment:
extra 👍 i forgot to comment here.

i would call it `name` and make a computed property as @jakewelton said.